bounties_api/std_bounties/client_helpers.py

```diff
@@ -103,7 +105,7 @@ def map_fulfillment_data(data_hash, bounty_id, fulfillment_id):
     except StatusError as e:
         if e.original.response.status_code == 504:
             redis_client.set('blacklist:' + str(bounty_id), True)
-            logger.error('Timeout for bounty id: ' + bounty_id)
+            logger.error('Timeout for bounty id: ' + str(bounty_id))
             raise e
```

Comment:
Same for this `raise e`

Reply:
@codeluggage definitely makes sense. I now understand what you were saying. Thanks :)